devices/xiaomi.js

```diff
@@ -1285,4 +1285,29 @@ module.exports = [
             await device.getEndpoint(1).write('aqaraOpple', {'mode': 1}, {manufacturerCode: 0x115f, disableResponse: true});
         },
     },
+    {
+        zigbeeModel: ['lumi.switch.b1nc01'],
+        model: 'QBKG40LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara E1 1 gang switch (with neutral)',
+        fromZigbee: [fz.on_off, fz.xiaomi_power, fz.xiaomi_multistate_action],
+        toZigbee: [tz.on_off, tz.xiaomi_power, tz.xiaomi_switch_operation_mode_opple, tz.xiaomi_switch_power_outage_memory],
+        endpoint: (device) => {
+            return {'system': 1};
```

Comment:
I don't think is needed, can you try removing it.

Reply:
@Koenkk done!